xsnippet/api/services/snippet.py

```diff
@@ -111,7 +129,11 @@ def __init__(self, db):
             ('created_at', sort['created_at']),
             ('_id', sort['_id']),
         ])
-        return await query.limit(limit).to_list(None)
+
+        snippets = await query.limit(limit).to_list(None)
+        for snippet in snippets:
+            snippet['content'] = snippet.pop('changesets', [])[-1]['content']
```

Comment:
nit: maybe we should leave this for the API layer, but we can discuss this later, when we actually start using the changesets

Reply:
I was thinking about it and agree it probably would be better this way.. I don't remember why I decided to put it here for now, but I bet there were a reason. I think we will change this place plenty of times while we really start using it :)